app/helpers/outdoor_routing_helper.rb

```diff
@@ -0,0 +1,40 @@
+require 'httparty'
+require 'json'
+
+module OutdoorRoutingHelper
+  def self.routing_url(start, dest)
+    "http://routing.openstreetmap.de/routed-foot/route/v1/driving/#{start[1]},#{start[0]};#{dest[1]},#{dest[0]}?overview=full&geometries=geojson"
+  end
+
+  def self.handle_outdoor_indoor_case(start, dest_building, res)
```

Comment:
Maybe it would be better to name this method similarly to the `route_xy` methods from the `IndoorRoutingHelper` as they seem to do very similar things.